src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,23 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<? extends Machine> get() {
+        return listOfBulldozers();
```

Comment:
It's not necessary to create a separate method for getting list of bulldozers

Reply:
It is not mistake?

Reply:
The method listOfBulldozers() is only called once, and the list of bulldozers is not being reused or modified elsewhere in the class, so it doesn't need to be abstracted into its own method. The code would be simpler and easier to read if the list was just created directly in the get() method